include/fmt/format.h

```diff
@@ -417,7 +417,7 @@ typedef basic_memory_buffer<char> memory_buffer;
 typedef basic_memory_buffer<wchar_t> wmemory_buffer;
 
 /** A formatting error such as invalid format string. */
-class format_error : public std::runtime_error {
+class FMT_API format_error : public std::runtime_error {
```

Comment:
Only the destructor needs to be marked with `FMT_API`.

Reply:
That was my first pull request, but then I got undefined reference to the vtable, which is not exported in that case. If you know how to export the vtable explicitly I will change it.

Reply:
Ah, OK then.